include/solvers/mpm_base.tcc

```diff
@@ -329,6 +329,36 @@ bool mpm::MPMBase<Tdim>::initialise_particles() {
                        particles_volume_end - particles_volume_begin)
                        .count());
 
+    // Get material sets
```

Comment:
Can you change the comment?

Reply:
updated to:
'''
// Material id update using particle sets
'''

Reply:
commit https://github.com/cb-geo/mpm/pull/674/commits/150a22ff9d328c9eeb7bce0a808324145b116746